manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -181,7 +181,8 @@ data:
           "enabled": true
         },
         "rewriteAppHTTPProbe": true,
-        "templates": {}
+        "templates": {},
+        "useLegacySelectors": false
```

Comment:
Do we want it false by default ?

Reply:
true by default in master (1.10), false in 1.9 was what we discussed. I will change to false in the backport